v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm. 
```

Comment:
It looks like several lines have white space at the end of the line, like this line.  Could you clean up any white space at the end of the lines you've changed or added.